src/Content/Text/BBCode.php

```diff
@@ -354,6 +354,9 @@ public static function getAttachedData($body, $item = [])
 				$post['url'] = $links[0][1];
 			}
 
+			// Simplify "video" element
+			$post['text'] = preg_replace("(\[video.*?\s?src\s?=\s?([^\s\]]*).*?\].*?\[\/video\\])ism", "[video]$1[/video]", $post['text']);
```

Comment:
I simplified the RegExp a little, can be used as is:
```suggestion
			$post['text'] = preg_replace('(\[video.*?\ssrc\s?=\s?([^\s\]]+).*?\].*?\[/video\])ism', '[video]$1[/video]', $post['text']);
```

Reply:
Okay. I really fought today. RegExps are a PITA ...

Reply:
You can defer them to me if you want, they're like a fun puzzle for me.